### ISSUE TITLE
Does not recognize tilde (~) in the path on Windows

### ISSUE THREAD

**Issue by balthild** [kingToolbox/WindTerm#464]
This path will cause an authentication failure:

![image](https://user-images.githubusercontent.com/2662758/150454488-d621140e-d3d9-484d-ba34-09300ba471b0.png)

But this will success:

![image](https://user-images.githubusercontent.com/2662758/150454582-820e7c9e-2088-4347-aa46-328c51a9a071.png)

**Comment by kingToolbox** [comment]
In `Windows`, `~` is just a character and has no special meaning. So the current result is reasonable, please use the absolute path to specify the identity file path, thank you.

**Comment by balthild** [comment]
But when I sync the profile between two computers with different OS, It will not work if the path is windows-style (C:\xxx).

Also, the tilde works in Microsoft PowerShell. So I think supporting tilde on Windows isn’t that unreasonable.

**Comment by kingToolbox** [comment]
Yes, `PowerShell` does support this. I will add support for this in the next `WindTerm_2.3.0_Prerelease_3` version so that the `Profiles` folder can be shared between different systems, but only when `~` is at the beginning of the path.

The new version is expected to be released this weekend, please stay tuned, thank you.

**Comment by kingToolbox** [comment]
I am happy to inform you that [WindTerm_2.3.0_Prerelease_3](https://github.com/kingToolbox/WindTerm/releases/tag/2.3-prerelease) has been released and this issue has been fixed, please download and check it, thank you.
